jobbergate-docs/src/developer_guide/s3_file_storage.rst

```diff
@@ -0,0 +1,57 @@
+=================
+ S3 File Storage
+=================
+
+Besides the information kept in the database, application and job-scripts files are
+stored in an S3 Bucket.
+
+The bucket is named according to the ``S3_BUCKET_NAME`` environment variable used by
+the Jobbergate API.
+
+Within the the bucket, there should be two top-level folders, and the structure should
+appear similar to:
+
+.. code-block::
```

Comment:
This is outdated. The supporting files changed the file structure. I'll create a task to cover this.